packages/squiggle-lang/src/fr/dist.ts

```diff
@@ -313,4 +324,67 @@ export const library: FRFunction[] = [
       ),
     ],
   }),
+  maker.make({
```

Comment:
These definitions probably should be in `fr/genericDist.ts`, since other dist operations are defined there. `fr/dist.ts` is mostly for sample set constructors.

Also, there's `distResultToValue` function there that could be used instead of `Result.getExt` to throw the correct `REDistributionError`.